engine/src/main/scala/cromwell/engine/backend/io/filesystem/gcs/GcsFileSystemProvider.scala

```diff
@@ -40,6 +40,17 @@ object GcsFileSystemProvider {
   // Then, at the end of this time, I decided to just pick some numbers arbitrarily.
   private val retryInterval = 500 milliseconds
   private val retryCount = 3
+
+  def withRetry[A](f: => A, retries: Int = retryCount): A = Try(f) match {
+    case Success(res) => res
+    case Failure(ex: GoogleJsonResponseException)
+      if retries > 0 &&
+        (ex.getStatusCode == 404 || ex.getStatusCode == 500) =>
+      // FIXME remove this sleep
```

Comment:
Hopefully soon - Fixing this is a bigger refactoring than this ticket which is just a "get this into the next release to reduce failure on transient google errors"